webapp/src/components/github_issue/index.tsx

```diff
@@ -21,11 +21,9 @@ const GithubIssue = ({theme, post}: GithubIssueProps) => {
 
     const updateStyleForCloseOrReopenButton = () => {
         if (postProps.status === 'Close') {
-            buttonClassName = 'btn btn-danger';
-            style.close_or_reopen_button.backgroundColor = '#dc3545';
             return style.close_or_reopen_button;
         }
-        return null;
+        return style.other_buttons;
     };
```

Comment:
We aren't doing anything here, simply returning the variables. We can get rid of this function if not needed.